### DISCUSSION TITLE
How to upgrade Next.js from 7.0.2 to latest

### DISCUSSION THREAD

**Discussion by sambhav2612** [vercel/next.js#11878] [Help] [answered]
Hi,

We have been using this very old version of Next.js for quite some time now and have been wanting to upgrade it to latest, given the documentation only supports upgrades from 8.x.x to latest, I had no resort but to post this query here since it isn't an issue to report.

Thanks!

**Comment by Timer** [comment, marked as answer]
Next.js 7 => 8 should be fully backwards compatible in most cases. There are no explicit upgrade instructions.

tl;dr only edge cases should break, which need handled on a case-by-case basis

Please see the relevant blog post:
https://nextjs.org/blog/next-8

**Reply by sambhav2612** [reply]
thanks!
